packages/components/src/interfaces/complex/Infobar.ts

```diff
@@ -8,23 +8,27 @@ export const InfobarSchema = Type.Object(
     type: Type.Literal("infobar", { default: "infobar" }),
     title: Type.String({
       title: "Title",
+      maxLength: 100,
```

Comment:
is there commonality between the various max lengths? e.g. all titles are max len of 100? If so,  I would prefer to extract these into a constants file

Reply:
I'll be going through them with @sehyunidaaa later but at present, alot of these weren't present on the excel and i had to make a judgement call, which is why they're left as-is. 

let me finalize them with sehyun and after that, we can extract them out!